File: 2609번 최대공약수와 최소공배수.py
def gcd(p, q):

	if q == 0:
		return p
	else:
		return gcd(q, p % q)

def lcm(a, b):

	return int(a * b / gcd(a, b))

a, b = map(int, input().split())

g = gcd(a, b)
l = lcm(a, b)

print( g )
print( l )

File: 2609번 최대공약수와 최소공배수.cpp
#include <iostream>
using namespace std;

int gcd(int p, int q)
{
	if( q == 0 )
		return p;
	else
		return gcd(q, p % q);
}

int lcm(int a, int b)
{
	return (a * b) / gcd(a, b);
}

int main(void)
{
	int a, b;
	
	scanf("%d", &a);
	scanf("%d", &b);
	
	int g = gcd(a, b);
	int l = lcm(a, b);
	
	cout<<g<<endl;
	cout<<l<<endl;
	
	return 0;
}
